bundle.yaml

```diff
@@ -20626,10 +20626,20 @@ rules:
   - ""
   resources:
   - services
-  - services/finalizers
   - endpoints
   verbs:
   - get
+  - list
```

Comment:
This is generated from https://github.com/prometheus-operator/prometheus-operator/blob/master/jsonnet/prometheus-operator/prometheus-operator.libsonnet#L58-L123. Changes first need to be done in jsonnet code and after that you need to run `make generate` to propagate this change into other generated files.

Reply:
also there's no need to grant list/watch permissions on services/endpoints to the prometheus-operator service account.